js/audio.js

```diff
@@ -1,48 +1,18 @@
-function listenKeypress() {
```

Comment:
this file is now so small is it worth having it? a http request just to get two functions that are only used from anims-missiles.js - which should be renamed also as it does a lot more than animate the missiles now

Reply:
addressed in 038d9af